build.js

```diff
@@ -42,6 +42,11 @@ process.on('unhandledRejection', (reason, p) => {
  * Virtual imports made available to all bundles. Used for site config and globals.
  */
 const virtualImports = {
+  webdev_analytics: {
```

Comment:
why put these here instead of in `webdev_config`?

Reply:
You don't need to import everything if you only want a subset of config. Importing `{* as foo}` can bring all values in, so it's nice to break it apart into different configs.